### ISSUE TITLE
[+] Provide Docker Images

### ISSUE THREAD

**Issue by Azathothas** [iglunix/iglunix#131]
Hi, is it possible to build a docker image for IgluLinux?
A docker image, like other distros, would enable people to quickly test/run IgluLinux on their system.
Thanks!

**Comment by aheirman** [comment]
I am sad to say I do not know "IgluLinux". Having up to date Iglunix images would be nice.

**Comment by Ella-0** [comment]
Yeah but we've got bigger problems. We need infrastructure sorting out. before this. Closing for now
